modules/apps/object/object-test/src/testFunctional/tests/ObjectRelationships.testcase

```diff
@@ -985,6 +985,133 @@ definition {
 		}
 	}
 
+	@description = "LPS-151676 - Verify that it's possible to relate a Custom Object entry with many Native Object entries (One-to-Many)"
+	@priority = "5"
+	test CanRelateOneToManyCustomObjectEntryWithNativeObjectEntries {
+		property portal.acceptance = "true";
+
+		task ("Add 3 users entries to relate to custom object entries") {
+			JSONUser.addUser(
+				userEmailAddress = "user1@liferay.com",
+				userFirstName = "userfn_1",
+				userLastName = "userln_1",
+				userScreenName = "usersn1");
+
+			JSONUser.setFirstPassword(
+				agreeToTermsAndAnswerReminderQuery = "true",
+				requireReset = "false",
+				userEmailAddress = "user1@liferay.com");
+
+			JSONUser.addUser(
+				userEmailAddress = "user2@liferay.com",
+				userFirstName = "userfn_2",
+				userLastName = "userln_2",
+				userScreenName = "usersn2");
+
+			JSONUser.setFirstPassword(
+				agreeToTermsAndAnswerReminderQuery = "true",
+				requireReset = "false",
+				userEmailAddress = "user2@liferay.com");
+
+			JSONUser.addUser(
+				userEmailAddress = "user3@liferay.com",
+				userFirstName = "userfn_3",
+				userLastName = "userln_3",
+				userScreenName = "usersn3");
+
+			JSONUser.setFirstPassword(
+				agreeToTermsAndAnswerReminderQuery = "true",
+				requireReset = "false",
+				userEmailAddress = "user3@liferay.com");
+		}
```

Comment:
We could use a loop here since we're adding three users with similar names, `usersnX`. It isn't necessary, but it's a suggestion.